src/App.jsx

```diff
@@ -1,6 +1,7 @@
 /* eslint-env browser */
 import React, { Component } from 'react';
 import Header from './components/header';
+import PromoHeader from './components/promoHeader';
```

Comment:
Totally sniffy, but maybe rename the file to be PromoHeader, to match the component name itself>